charts/xrd-vrouter/values.yaml

```diff
@@ -244,30 +244,33 @@ cpu: {}
   # appropriately between the control-plane and dataplane.
   #controlPlaneCpuset: ""
   #dataPlaneCpuset: ""
-  # HyperThreading mode.
-  # This option must be used when the CPU has hyperthreading turned on
-  # in order to ensure the dataplane has whole physical CPU cores and prevent
-  # 'noisy neighbor' issues.
-  # This mode supports two values:
-  #  - 'off' - no hyperthreading (default)
-  #  - 'pairs' - where hyperthreaded siblings are consecutive, i.e.
-  #               * physical core 0 has logical cores 0 and 1
-  #               * physical core 1 has logical cores 2 and 3
-  #               * etc...
-  #hyperThreadingMode: "off"
   #
   # Deprecated cpu settings:
   #
   # Override the cpuset to use for XRd. Must not be used in conjunction with
   # controlPlaneCpuset and dataPlaneCpuset.
-  # Deprecated. Use controlPlaneCpuset and dataPlaneCpuset instead.
   # This must be a subset of the CPUs available for XRd to run on.
+  # Deprecated. Use controlPlaneCpuset and dataPlaneCpuset instead.
   #cpuset: ""
   # Override the number of CPUs used for control-plane threads from the pool of
   # CPUs in the XRd cpuset. Must not be used in conjunction with
   # controlPlaneCpuset and dataPlaneCpuset.
   # Deprecated. Use controlPlaneCpuset and dataPlaneCpuset instead.
   #controlPlaneCpuCount: 2
+  # HyperThreading mode.
+  # This option must be used when the CPU has hyperthreading turned on
+  # in order to ensure the dataplane has whole physical CPU cores and prevent
+  # 'noisy neighbor' issues.
+  # This mode supports two values:
+  #  - 'off' - no hyperthreading (default)
+  #  - 'pairs' - where hyperthreaded siblings are consecutive, i.e.
+  #               * physical core 0 has logical cores 0 and 1
+  #               * physical core 1 has logical cores 2 and 3
+  #               * etc...
+  # Must not be used in conjunction with controlPlaneCpuset and
+  # dataPlaneCpuset.
+  # Deprecated. Use controlPlaneCpuset and dataPlaneCpuset instead.
```

Comment:
As discussed, please check with Paul if this should be deprecated.

In particular - specifying neither `controlPlaneCpuset` or `dataPlaneCpuset` is fine and not deprecated.  It should be fine to specify neither, and also `hyperThreadingMode`.  Should this be deprecated?

If not, then the correct change is to implement the mutual-exclusivity check (and document this), but not move `hyperThreadingMode` to deprecated.

Reply:
Summary of offline discussion:
* hyperThreadingMode should not be deprecated as it is useful when not specifying any cpuset and in the future we may wish to allow it to be used alongside CP/DP_CPUSET env vars.
* For the time being it can't be used alongside CP/DP_CPUSET env vars so comment and test for this in the Helm.